examples/use-cases/cloud_router_aws/main.tf

```diff
@@ -425,21 +399,13 @@ resource "aws_cloud_router_connection" "crc_1" {
   zone           = var.pf_crc_zone1
   is_public      = var.pf_crc_is_public
   speed          = var.pf_crc_speed
-  depends_on = [
-    cloud_router.cr
-  ]
   lifecycle {
-    ignore_changes = [
-      circuit_id,
-      description,
-      pop,
-      zone
-    ]
+    ignore_changes = all
```

Comment:
Again, why should we ingore all changes here? How will we detect that we are out of sync with PacketFarbirc?

Reply:
removed the ignore_changes = all